modules/imgcodecs/src/ios_conversions.mm

```diff
@@ -116,6 +118,8 @@ void UIImageToMat(const UIImage* image,
         if (!alphaExist)
             bitmapInfo = kCGImageAlphaNoneSkipLast |
                                 kCGBitmapByteOrderDefault;
+        else
+            m = cv::Scalar(0);
```

Comment:
I didn't actually time the cost of setting all those bytes to zero on a mobile device, but since 3/4 of the work is duplicated in the draw call (which sets RGB), it seemed wasteful.  We could initialize just the alpha channel to zero I suppose, if that's more efficient (misaligned data access may actually make this more costly on some processors/architectures).